apis/python/src/tiledbsoma/options/_soma_tiledb_context.py

```diff
@@ -66,25 +70,30 @@ class SOMATileDBContext(ContextBase):
 
     def __init__(
         self,
-        tiledb_config: Optional[Dict[str, Union[str, float]]] = None,
-        timestamp: Optional[OpenTimestamp] = None,
-        threadpool: Optional[ThreadPoolExecutor] = None,
+        tiledb_ctx: "tiledb.Ctx" | None = None,
+        tiledb_config: Dict[str, Union[str, float]] | None = None,
```

Comment:
Please move `tiledb_ctx` after `tiledb_config`, here & in the docstring. The config-dict needs to be clearly "plan A".